package.json

```diff
@@ -73,6 +73,7 @@
     "@dnd-kit/core": "^6.1.0",
     "@dnd-kit/modifiers": "^7.0.0",
     "@dnd-kit/sortable": "^8.0.0",
+    "@emotion/cache": "^11.14.0",
```

Comment:
We need to configure emotion to inject the style elements inside our shadow root, and that requires us to configure a custom cache.